app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  validates :body, null: false
```

Comment:
Validating for null: false will not prevent a user from creating a comment with empty strings ("") body. Validate for `presence: true` instead.

Reply:
Okay, thank you very much for your feedback!